spark/v3.1/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestSetCurrentSnapshotProcedure.java

```diff
@@ -252,5 +252,60 @@ public void testInvalidRollbackToSnapshotCases() {
         IllegalArgumentException.class,
         "Cannot handle an empty identifier",
         () -> sql("CALL %s.system.set_current_snapshot('', 1L)", catalogName));
+
+    AssertHelpers.assertThrows(
```

Comment:
we don't want to introduce more usages of `AssertHelpers`, so better to use `Assertions.assertThatThrownBy` here